docs/source/conceptual_guides/lora.mdx

```diff
@@ -43,6 +43,22 @@ While LoRA is significantly smaller and faster to train, you may encounter laten
 
 This works because during training, the smaller weight matrices (*A* and *B* in the diagram above) are separate. But once training is complete, the weights can actually be merged into a new weight matrix that is identical.
 
+## Utils for LoRA
```

Comment:
Would it be useful to document `merge_and_unload` here is well?

Reply:
It is explained just above this

Reply:
I see, thanks